site/content/partner-program-api/index.md

```diff
@@ -526,247 +659,477 @@ curl --request GET \
+* `401 - Unauthorized` - missing or incorrect authorization header
+
+
 # Solution Partner
 
 ## Create License
-`POST /v2/partners/resellers/licenses` - create new license for a client
-
-#### Headers
-* `Authorization` - **required** - `Bearer <YOUR_API_TOKEN>`
-
-#### Payload
-* `client_name` - **required** - client's full name
-* `client_email` - **required** - client's email address
-* `purchase_order` - optional - custom parameter
 
-#### Response
-* `201 - Created` - example:
+>Response example:
 
 ```json
 {
     "license": "123",
     "client_name": "Joe Doe",
     "client_email": "joe@example.com",
-	"purchase_order": "Test license",
-	"creation_timestamp": "2018-03-06 12:59:13",
-	"end_timestamp": "2018-04-05 11:59:12",
-    "cc_added": false,
-    "paid": false,
+    "purchase_order": "Test license",
+    "creation_timestamp": "2018-03-06 12:59:13",
+    "end_timestamp": "2018-04-05 11:59:12",
+    "cc_added": true,
+    "paid": true,
     "seats": 100,
     "plan": "team",
     "billing_cycle": "monthly",
-	"total_discount": 0,
-    "discount_percent": 20,
+    "total_discount": 0,
+    "discount_percent": 0,
     "expired": false,
     "blocked": false,
+    "payment_origin": "client",
+    "conversations": 403,
+    "conversations_last_30_days": 195,
+    "total_commission": 125,
+    "commission_percent": 20,
+    "data_center": "fra",
     "recurly": {
         "subscriber": false,
         "in_trial": true,
         "recurrent": false
     }
 }
 ```
-* `400 - Bad Request` - missing or incorrect parameters
-* `401 - Unauthorized` - missing or incorrect authorization header
-* `409 - Conflict` - given `client_email` already has a LiveChat license
-
 
-## Get License (Client) List
-`GET /v2/partners/resellers/licenses` - get license list
+`POST /v2/partners/resellers/licenses` - **DEPRECATED**
+`POST /v2/partners/solutions/licenses` - create new license for a client
 
 #### Headers
 * `Authorization` - **required** - `Bearer <YOUR_API_TOKEN>`
 
-#### Query parameters
-* `date_from` - optional - get report for licenses created from a given date
-* `date_to` - optional - get report for licenses created to a given date
+#### Payload
+* `client_name` - **required** - client's full name (min. 5, max. 100 characters)
+* `client_email` - **required** - client's email address
+* `payment_origin` - **required** - subscription's payment management, one of `partner` or `client`
+* `trial_duration` - optional - days of trial period (min. 14, max. 60)
+* `coupon_id` - optional - coupon id (applied when `payment_origin="client"`)
+* `data_center` - optional - your client's data storage center, one of `dal` or `fra`
+* `purchase_order` - optional - custom parameter
 
-**Note:** Date format: `YYYY-MM-DD`
+**Note:** Data center: `dal` - Dallas, USA, `fra` - Frankfurt, Germany. Default value of `data_center` is `dal`. Trial duration for `payment_origin="partner"` is set to 14-days, for `payment_origin="client"` default vaule is 14-days.
```

Comment:
**Note:** Data center: `dal` - Dallas (US), `fra` - Frankfurt (EU). Default value of `data_center` is `dal`.

^ That would be enough